### ISSUE TITLE
culturefeed_search_item_load_by_title() function does not use its second parameter type

### ISSUE THREAD

**Issue by michel-gr** [cultuurnet/culturefeed#109]
In the helpers.inc file the following function

```
function culturefeed_search_item_load_by_title($title, $type) {

  $search_string = trim($title);
  $parameters = array();
  $parameters[] = new \CultuurNet\Search\Parameter\Query('title:"' . $search_string . '"');
  $parameters[] = new \CultuurNet\Search\Parameter\FilterQuery('type:actor');
  $parameters[] = new \CultuurNet\Search\Parameter\Group();
  $result = culturefeed_get_search_service()->search($parameters);

  if ($result->getTotalCount()  > 0) {
    $items = $result->getItems();
    return $items[0];
  }

  return FALSE;

}
```

defines a $type which is never used, it inserts actor hard coded into the FilterQuery.

**Comment by stijnswaanen** [comment]
solved: https://github.com/cultuurnet/culturefeed/commit/c3c9f913eea2b0f0938c5e7039e763e565c42f67

**Comment by zuuperman** [comment]
This is a wrong fix. The type should be inserted in the filter query instead of the hardcoded type:actor

**Comment by stijnswaanen** [comment]
fixed again: #178
